vulnfeeds/cves/versions.go

```diff
@@ -707,7 +729,53 @@ func Commit(u string) (string, error) {
 	return "", fmt.Errorf("Commit(): unsupported URL: %s", u)
 }
 
-// For URLs referencing commits in supported Git repository hosts, return an AffectedCommit.
+// Some Git hosting has different human-accessible URLs for Git web interfaces to what is cloneable by Git and requires translation.
+// e.g.
+// the cGit-hosted repo:
+//
+//	https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git [web browseable]
+//	https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git [cloneable]
+//
+// the cGit-hosted repo:
+//
+//	https://git.savannah.gnu.org/cgit/emacs.git [web browseable]
+//	https://git.savannah.gnu.org/git/emacs.git [cloneable]
+//
+// the GitWeb-hosted repo:
+//
+//	https://git.gnupg.org/cgi-bin/gitweb.cgi?p=libksba.git [web browseable]
+//	git://git.gnupg.org/libksba.git [cloneable]
+func maybeRemapRepository(repo string, link string) (remappedRepo string) {
```

Comment:
Is this function no longer required?

Reply:
Oh yeah, thanks, no it's not, I'll remove it.

Reply:
Removed in d36a83b